src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_commitfilelist.yaml

```diff
@@ -0,0 +1,114 @@
+post:
+  summary: Commit package changes.
```

Comment:
`Commit package changes.` is the same description as for the `POST
/source/{project_name}/{package_name}?cmd=commit` endpoint. Do they both perform the same task? Can we modify the summary to make it easier to know what is each endpoint for?

![Screenshot from 2023-03-10 14-00-47](https://user-images.githubusercontent.com/24919/224322567-d0c24733-c254-49a7-94fc-0b5064044e83.png)